doc/source/user_guide/quantity.rst

```diff
@@ -29,6 +29,21 @@ through multiplication
     meter = ansunits.Quantity(value=1, units=ureg.m)
     meter = 1 * ureg.m
 
+With ``numpy`` installed, a ``Quantity`` can be created using lists or np.array.
```

Comment:
- In text, we should write NumPy (as they do in their documentation)
- Also in text, we should write NumPy arrays (again, as they do)